docs/usestate-hook.md

```diff
@@ -29,6 +34,80 @@ let make = (~initialCount) => {
     <button onClick={_ => setCount(prevCount => prevCount + 1)}>
       {React.string("+")}
     </button>
-  </>;
+  </section>;
 };
 ```
+
+## Using Event values with useState
```

Comment:
this is duplicated from `docs/usestate-event-value.md`. are you planning to delete that page?